engine/src/main/java/org/terasology/rendering/dag/states/StateValue.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.states;
+
+/**
+ *
+ */
+public enum StateValue {
+    ENABLED, // state change must happen
+    OPTIONAL,
+    DISABLED,
+}
```

Comment:
Scrap previous comment. In line with earlier comments this is no longer relevant as I wouldn't encapsulate values in their own instance but directly in an instance of a class implementing StateChange.